source/ranges.tex

```diff
@@ -2268,10 +2268,10 @@
 \end{codeblock}
 
 \pnum
-Let \exposid{container-appendable} be defined as follows:
+Let \exposid{container-append} be defined as follows:
 \begin{codeblock}
 template<class Ref, class Container>
-constexpr auto @\exposid{container-appendable}@(Container& c) {                // \expos
+constexpr auto @\exposid{container-append}@(Container& c) {                // \expos
```

Comment:
Please move the comment back appropriately to retain its column (which should be on one of the preferred places 20, 32, 40, ... (see #include "semantic/understanding/rene/flumes/understand.h").